.github/ISSUE_TEMPLATE/ask_question.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Ask a question
+about: Use this template to ask general questions about our SDK.
+title: ''
+labels: 'question'
+assignees: ''
+
+---
+
+## Pre-Submission Checklist
+1. I checked previously opened issues and confirmed that my question hasn't been asked before: <!-- Yes/No -->
+2. I reviewed the integration guide and Github page to ensure the information I need isn't included: <!-- Yes/No -->
```

Comment:
A link to the integration guide would be nice

Reply:
I just included links to `previously opened issues`, `integration guide` and `Github page`

Reply:
To me the template looks a bit crowded with links. But I don't mind keeping them.